src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -32,37 +58,86 @@
+                               :borderBottom (str "1px solid " (:line-gray style/colors))
+                               :padding "20px 48px 18px"
+                               :fontSize "137%" :fontWeight 400 :lineHeight 1}}
+                 "Import a Method Configuration"]
+                [:div {:style {:padding "22px 48px 40px"
+                               :backgroundColor (:background-gray style/colors)}}
+                 "Destination Name : "
+                 (style/create-text-field
+                   {:defaultValue (get-in @state [:selected-conf "name"])
+                    :ref "destinationName"})
+                 [:br]
+                 "Destination Namespace : "
+                 (style/create-text-field
+                   {:defaultValue (get-in @state [:selected-conf "namespace"])
+                    :ref "destinationNamespace"})
+                 [:br]
+                 [comps/Button
+                  {:title-text "Import configuration"
+                   :icon :plus
+                   :onClick
+                   (fn []
+                     (let [workspace (:workspace props)
+                           dest-name-value (.-value
+                                             (.getDOMNode (@refs "destinationName")))
+                           dest-namespace-value (.-value (.getDOMNode
+                                                           (@refs "destinationNamespace")))
+                           dest-workspace-name (get workspace "name")
+                           dest-workspace-namespace (get workspace "namespace")
+                           methodRepoQuery (create-post-data
+                                             (:selected-conf @state)
+                                             dest-name-value
+                                             dest-namespace-value
+                                             dest-workspace-name
+                                             dest-workspace-namespace)
+                           url-method :post
+                           url-path (create-path
+                                      dest-workspace-namespace
+                                      dest-workspace-name)
+                           url-data (utils/->json-string methodRepoQuery)
+                           url-canned-response {:status 201
+                                                :delay-ms (rand-int 2000)}
+                           url-on-done-func (fn [{:keys [success? xhr]}]
+                                              (if success?
+                                                (utils/rlog  (str "success in import! "
+                                                               "Here, perhaps trigger re-render "
+                                                                "of the MCs in the current "
+                                                                "workspace?"))
+                                                (let [returned-err-msg
```

Comment:
Why not now?  The refactoring story is for when structure starts getting out of hand, not for fixing simple things.